docs/source/new-docs/getting-started/system-requirements/index.rst

```diff
@@ -8,30 +8,84 @@
++==================+=========================================================+
+| Linux            | RedHat 8, AlmaLinux 8, RockyLinux 8 equivalent or later |
++------------------+---------------------------------------------------------+
+|                  | Debian 10 equivalent or later                           |
++------------------+---------------------------------------------------------+
+| MacOS            | 12 or later *(not officially supported)*                |
++------------------+---------------------------------------------------------+
+| Windows          | 10 or later *(not officially supported)*                |
++------------------+---------------------------------------------------------+
+
+At the moment, the installation instructions for GeoIPS (which you can find [HERE])
+work for these OS/processor archetype combos:
+
++---------------------------+--------------------+--------------------+--------------------+------------------------+
+|                           | Conda              | Expert Install     | Docker (native)    | Docker (x86 emulation) |
++===========================+====================+====================+====================+========================+
+| Debian (x86)              | ✅                 | ✅                 | ✅                 | N/A                    |
++---------------------------+--------------------+--------------------+--------------------+------------------------+
+| Redhat (x86)              | ✅                 | ❓                 | ❓                 | N/A                    |
++---------------------------+--------------------+--------------------+--------------------+------------------------+
+| Mac (x86)                 | ❓                 | ❓                 | ❓                 | N/A                    |
++---------------------------+--------------------+--------------------+--------------------+------------------------+
+| Mac (arm)                 | ❌                 | ❓                 | ❌                 | ✅                     |
++---------------------------+--------------------+--------------------+--------------------+------------------------+
+| Windows with WSL2 (x86)   | ❓                 | ❓                 | ❓                 | N/A                    |
++---------------------------+--------------------+--------------------+--------------------+------------------------+
+
+User System Requirements
+------------------------
+
++----------+-------------+-------------+--------------------------------------+
+| Hardware | Minimum     | Recommended | Comments                             |
++==========+=============+=============+======================================+
+| CPU      | 2 CPU       | 4 CPU       |                                      |
++----------+-------------+-------------+--------------------------------------+
+|| Memory  || 12GB       || 128GB      || Enough for most GeoIPS imagery.     |
```

Comment:
This was minimum 16GB in the SRS, I think 16GB is safer.  Did you calculate 12 GB from ABI infrared?

Reply:
Considering recommended is 128 GB, 12GB minimum is probably fine (since clearly that won't run EVERYTHING)